dashboard/views/_general/_position.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import render
+
+from dashboard.models import Position
+
+
+def positions(request):
+    """ Renders all of the positions currently in the chapter """
+    # doing a .all() ensures that the positions are listed in order of succession since they're created in that order

```

Comment:
Should this have `positions = Position.objects.all()` ? If not, you can remove the comment as to why the .all() method is included.